src/elements/content-sidebar/activity-feed/comment/BaseComment.scss

```diff
@@ -3,28 +3,51 @@
 $repliesPaddingLeft: 40px;
 
 .bcs-BaseComment {
-    .bcs-Comment-timestamp {
-        margin-bottom: $bdl-grid-unit * 4;
-    }
-
-    .bcs-ActivityTimestamp {
-        color: $bdl-gray-65;
-    }
+    font-size: $bdl-fontSize--dejaBlue;
 
     .bcs-CommentForm-input {
         background-color: $white;
     }
 
+    .bcs-CommentFormControls {
+        display: flex;
+        flex-direction: row-reverse;
+
+        .btn {
+            margin: $bdl-grid-unit * 3 0 0 $bdl-grid-unit * 3;
+        }
+    }
+
     .btn-plain:focus-visible {
         outline: auto;
     }
 
-    .bcs-Replies {
-        margin: $bdl-grid-unit * 3 0;
-        padding-left: $repliesPaddingLeft;
+    .bcs-Comment-media {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        width: 100%;
+        margin-bottom: $bdl-grid-unit * 4;
+    }
+
+    .mention-selector-wrapper {
+        margin: 0;
     }
 
-    .bcs-Comment-media {
-        margin-bottom: $bdl-grid-unit * 3;
+    .public-DraftEditor-content {
+        width: 100%;
+        min-height: $bdl-grid-unit * 12;
     }
 }
+
+.bcs-BaseComment-header {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    padding: 2;
```

Comment:
❓ Do we need a unit here?

Reply:
Apparently not, haha! Lemme look into this. Good catch.

Reply:
Comparing to designs, as far as I can tell, that shouldn't be there. Doesn't seem to be any padding that's unique to the header (although it's kinda hard to tell for sure...). I'll remove that 👍